backend/oasst_backend/models/user_stats.py

```diff
@@ -40,14 +41,27 @@ class UserStats(SQLModel, table=True):
     accepted_replies_assistant: int = 0
     accepted_replies_prompter: int = 0
 
-    reply_assistant_ranked_1: int = 0
-    reply_assistant_ranked_2: int = 0
-    reply_assistant_ranked_3: int = 0
-
-    reply_prompter_ranked_1: int = 0
-    reply_prompter_ranked_2: int = 0
-    reply_prompter_ranked_3: int = 0
+    reply_ranked_1: int = 0
+    reply_ranked_2: int = 0
+    reply_ranked_3: int = 0
 
     # only used for time span "total"
     streak_last_day_date: Optional[datetime] = Field(nullable=True)
     streak_days: Optional[int] = Field(nullable=True)
+
+    def compute_leader_score(self) -> int:
+        return (
+            self.prompts
+            + self.replies_assistant * 4
```

Comment:
how about making these constants into settings?

Reply:
We need to find a "stable" formula for the leader-score .. calculation is "magic" right now .. the exact way how to calculate it needs more discussion, e.g. could be based on acceptance rations or include negative values..